configure.ml

```diff
@@ -55,70 +58,70 @@ let opam_var v =
   let _ = Unix.close_process_in ch in
   s
 
+let concat = Filename.concat
 
 (* Compute actual values for config options *)
 let () =
   let prefix_set = !prefix <> "" in
   update "prefix" prefix (fun () -> opam_var "prefix");
   update "libdir" libdir (fun () ->
-      if prefix_set
-      then Filename.concat !prefix "lib"
-      else opam_var "lib");
+      if prefix_set then concat !prefix "lib" else opam_var "lib");
   update "mandir" mandir (fun () ->
-      if prefix_set
-      then Filename.concat !prefix "man"
-      else opam_var "man");
+      if prefix_set then concat !prefix "man" else opam_var "man");
   ()
 
 (* Output config options into lib/util/config.ml *)
 let () =
-  let f = Filename.concat (Filename.concat "lib" "util") "config.ml" in
+  let f = concat "src" @@ concat "lib" @@ concat "util" "config.ml" in
   let () = Format.printf "Generating file %s..." f in
   let ch = open_out f in
   let fmt = Format.formatter_of_out_channel ch in
-  let () = Format.fprintf fmt
-      "(* Static configuration, automatically generated by configure.ml *)@." in
+  let () =
+    Format.fprintf fmt
+      "(* Static configuration, automatically generated by configure.ml *)@."
+  in
   let () = Format.fprintf fmt {|let libdir = "%s"@.|} !libdir in
   let () = Format.fprintf fmt {|let mandir = "%s"@.|} !mandir in
-  let () = Format.fprintf fmt {|
-(* Dynamic configuration, relative to the executable path *)
-
-let follow dir path =
-  Filename.concat path dir
-
-let abs_exe_path =
-  let exe_name = Sys.executable_name in
-  if not (Filename.is_relative exe_name) then exe_name
-  else begin
-    let cwd = Sys.getcwd () in
-    Filename.concat cwd exe_name
-  end
-
-let datadir =
-  abs_exe_path
-  |> Filename.dirname
-  |> follow Filename.parent_dir_name
-  |> follow "share"
-  |> follow "alt-ergo"
-
-let pluginsdir = datadir |> follow "plugins"
-
-let preludesdir = datadir |> follow "preludes"
-
-|} in
+  let () =
+    Format.fprintf fmt
+      "@[<v 0>(* Dynamic configuration, relative to the executable path *)@,\
+       @,\
+       @[<v 2>let follow dir path =@,\
+       Filename.concat path dir@]@,\
+       @,\
+       @[<v 2>let abs_exe_path =@,\
+       let exe_name = Sys.executable_name in@,\
+       if not (Filename.is_relative exe_name) then exe_name@,\
+       @[<v 2>else begin@,\
+       let cwd = Sys.getcwd () in@,\
+       Filename.concat cwd exe_name@]@,\
+       end@]@,\
+       @,\
+       @[<v 2>let datadir =@,\
+       abs_exe_path@,\
+       |> Filename.dirname@,\
+       |> follow Filename.parent_dir_name@,\
+       |> follow \"share\"@,\
+       |> follow \"alt-ergo\"@]@,\
+       @,\
+       let pluginsdir = datadir |> follow \"plugins\"@,\
+       @,\
+       let preludesdir = datadir |> follow \"preludes\"@]@."
+  in
```

Comment:
Please revert his change, it makes the printed code much much less readable for no real improvement.